src/status_im/contexts/chat/home/view.cljs

```diff
@@ -131,7 +131,7 @@
   (let [scroll-ref     (atom nil)
         set-scroll-ref #(reset! scroll-ref %)]
     (fn []
-      (let [{:keys [universal-profile-url]} (rf/sub [:profile/profile])
+      (let [{:keys [universal-profile-url]} (rf/sub [:profile/universal-profile-url])
```

Comment:
Quick one: the new sub returns the string itself, so there's no need to destructure anymore.